src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -263,6 +285,21 @@ func (q *qemuAmd64) appendProtectionDevice(devices []govmmQemu.Device, firmware,
 				CBitPos:         cpuid.AMDMemEncrypt.CBitPosition,
 				ReducedPhysBits: cpuid.AMDMemEncrypt.PhysAddrReduction,
 			}), "", nil
+	case snpProtection:
+		return append(devices,
+			govmmQemu.Object{
+				Type: govmmQemu.SNPGuest,
+				ID:   "snp",
+				// See Guest Policy described in ABI:
+				// https://www.amd.com/system/files/TechDocs/56860.pdf
+				// Current setting: guest activation on multiple sockets allowed, no migration agent,
+				//                  no debug, SMT allowed, minimum ABI_MAJOR and ABI_MINOR set to 0
+				Policy:          0x30000, // hardcoded for now
```

Comment:
It might make sense to omit it from the PR to main, but then make a separate PR to the CC branch (once it has been rebased) that adds support for setting the policy. That would mirror what we do for SEV.

Reply:
I updated the code so that the policy isn't in this PR any longer.